### PR TITLE
Module 3

### PR DESCRIPTION
**Pull request by crangelsmith**
Fixes issue #19 

# Content
- [x] Introduction #20 
- [x] Rules of the game #21 
- [x] Atlas of visualisations #22 
- [x] Story telling #23 
- [x] Data exploration #24 
- [ ] Hands on

# Proofread and ready for external review 

- [ ] Introduction #20 
- [ ] Rules of the game #21 
- [ ] Atlas of visualisations #22 
- [ ] Story telling #23 
- [ ] Data exploration #24 
- [ ] Hands on

### CODE REVIEW

**Comment by crangelsmith** [comment]
I would like to add [this figure](https://www.economist.com/sites/default/files/images/print-edition/20210904_FBC105.png) to the context discussion of 3.4. Currently, we have only one figure talking about context but I feel is not representative of all these issues. 

The problem with this figure is that shows a sharp trend of something that is very small, people can overreact if they don't compare it with the number of universities and invited speakers there are usually on a single year.

File: coursebook/modules/m3/3.1-FiguresGoneWrong.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Here we've collated examples of figures that do a poor job of communicating the data. As we go through these examples we will have a brief discussion about each figure. Once you've had a chance to look at an example you can expand our notes below each example.
+
+## Example 1
+
+![](https://i.imgur.com/u16vTqy.png)
+<font size="1">[Source](https://www.callingbullshit.org/tools/tools_misleading_axes.html)</font>
+
+<details>
+    <summary>Example 1 notes</summary>
+This figure is a classic example of misleading information. The axes does not start at zero so the effect is visually exaggerated, and we are not given the uncertainty around these averages. The reader is therefore unable to interpret the practical significance of any differences between groups.
```
Comment (gmingas) [inline coursebook/modules/m3/3.1-FiguresGoneWrong.md:16]: I did not expect to see uncertainty around the averages. I assume these are numbers are averages of the actual working hours of the whole population, not a sample?

File: coursebook/modules/m3/3.1-FiguresGoneWrong.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+<details>
+    <summary>Example 3 notes</summary>
+Example 3 is very confusing because the _scale_ of the x-axis is altered mid-plot. Even after reading the caption it is very difficult to get a feel for the data because the grid-lines give a powerful signal of uniformity. 
+</details>
+
+## Example 4
+![](https://i.imgur.com/9lu7jVP.png)
+<details>
+    <summary>Example 4 notes</summary>
+The phenomenon that this xkcd comic is getting at is _Normalisation_. Normalisation is where you alter a scale to be between zero and one (usually). Failrue to normalise is when different measurements that themselves have different scales are plotted on the same scale. The comic shows the classic failure to normalise measurements by state population. A variable that was consistently 10% of the population would appear to vary across states.
```
Comment (gmingas) [inline coursebook/modules/m3/3.1-FiguresGoneWrong.md:49]: Also related to correlation vs. causation and confounders though not necessary to write this in the text

File: coursebook/modules/m3/hands-on.ipynb [hunk 3]
```diff
@@ -2,9 +2,34 @@
...
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "# Hands-on"
+    "## Hands-on Session\n",
+    "\n",
+    "Practice building figures.\n",
+    "\n",
+    "Explore the world in data dataset. You are free to use the full dataset (`df`), which has entries for many dates, or the aggregated dataset which only have the totals (`df_totals`).\n",
```
Comment (gmingas) [inline coursebook/modules/m3/hands-on.ipynb]: Do we want to use the whole hands-on session for this or should we ask them to explore the EQLS dataset in this time, to continue the hands-on session flow? I think they could be asked to do what currently happens in 3.5 (with guiding questions maybe) and we can use the 3.5 stuff as a guide for instructors?

**Review by gmingas — COMMENTED** [review]
Very good and well-written material overall and covers many topics. I fixed some typos and these are some comments on the notebooks:
- 3.2: Maybe do not add material on uncertainty given that there is already a lot in this notebook and others to cover the available time.
- 3.3: We could mention more explicitly the criticisms to Tufte made in Data Feminism and also question the validity of the ink ratio, etc: https://data-feminism.mitpress.mit.edu/pub/5evfe9yd/release/5#nao8xhq0duk
- 3.5: This could be moved to the hands-on session - also see separate comment.

**Comment by jack89roberts** [comment]
Hi here, I was trying to build the book from this branch and got an error due to not having the `clase25datos` kernel used in `3.5-DataVisForExploration.ipynb`,  could it be set to a default `Python 3 (ipykernel)` like the others?

**Comment by crangelsmith** [comment]
Hi @jack89roberts, yes that is my fault, I was piggybacking from one of my existing kernels but shouldn't have committed it. We should definitely move it to the default.

**Pull request merged**